src/config.js

```diff
@@ -20,6 +21,10 @@ const MAPPED_STATUS_COMMANDS = {
   'pr-merged-in-dev': '✅',
 };
 
+const PR_TEMPLATE_ALLOWED_CHANNELS = parseAllowedChannels(
+  process.env.PR_TEMPLATE_ALLOWED_CHANNELS
+);
+
```

Comment:
Group it in a feature as the other are please, if there is not a group for this feature (PR_TEMPLATE), then create a new group with the variables that are needed for that feature. This would help in further configurations 😄

Reply:
Completed